rfcs/0005-pablo-distribution.md

```diff
@@ -0,0 +1,667 @@
+
+Existing <img src="0005-pablo-distribution-assets/images/stem-39e8c7852cdbd74b28d331353778e128.png" width="21" height="9" alt="stem 39e8c7852cdbd74b28d331353778e128" /></span>
+staker reward,
+
+<img src="0005-pablo-distribution-assets/images/stem-e1359ae7d0fae29ebf9e42efcaa5536e.png" width="111" height="18" alt="stem e1359ae7d0fae29ebf9e42efcaa5536e" /></span>
+
+##### 5.3.1.1. When adding a new staker <span class="image"><img src="0005-pablo-distribution-assets/images/stem-0b46f732c83c0e66067b0e50c2156089.png" width="29" height="8" alt="stem 0b46f732c83c0e66067b0e50c2156089" /></span>, existing stakers(<span class="image"><img src="0005-pablo-distribution-assets/images/stem-55a049b8f161ae7cfeb0197d75aff967.png" width="9" height="6" alt="stem 55a049b8f161ae7cfeb0197d75aff967" /></span>) reward would be,
+
+<img src="0005-pablo-distribution-assets/images/stem-569c4bf984a23f18046277fd561e89a3.png" width="126" height="20" alt="stem 569c4bf984a23f18046277fd561e89a3" /></span>
+
+As this is less than what is expected above, an adjustment " delta P "
+to total reward pool can be made to allow realtime reward calculations,
+
+<img src="0005-pablo-distribution-assets/images/stem-8e958a64c877dcda40b652878c6c6768.png" width="119" height="20" alt="stem 8e958a64c877dcda40b652878c6c6768" /></span>
+
+<img src="0005-pablo-distribution-assets/images/stem-38d917fea7c6a7a47eb1aa77edd4da97.png" width="169" height="20" alt="stem 38d917fea7c6a7a47eb1aa77edd4da97" /></span>
+
+<img src="0005-pablo-distribution-assets/images/stem-5fcfbc0bc69ee8b8f356ce2bbfb42002.png" width="190" height="22" alt="stem 5fcfbc0bc69ee8b8f356ce2bbfb42002" /></span>
+
+<img src="0005-pablo-distribution-assets/images/stem-e1359ae7d0fae29ebf9e42efcaa5536e.png" width="111" height="18" alt="stem e1359ae7d0fae29ebf9e42efcaa5536e" /></span>
+
+**Therefore, the existing staker receives the same reward as before**
+
+To compensate for this new adjustment, a reduction <img src="0005-pablo-distribution-assets/images/stem-7c4ec4f9c189cb8f3edb39740e43c33f.png" width="16" height="10" alt="stem 7c4ec4f9c189cb8f3edb39740e43c33f" /></span>
+of reward for each staker needs to be tracked,
+
+<img src="0005-pablo-distribution-assets/images/stem-828ec270409cb6ff5cfc583587d0eae9.png" width="142" height="18" alt="stem 828ec270409cb6ff5cfc583587d0eae9" /></span>
+← (1)
+
+In general,
+
+<img src="0005-pablo-distribution-assets/images/stem-0f2f030a4f8a3c172e968af2768a3ec8.png" width="349" height="11" alt="stem 0f2f030a4f8a3c172e968af2768a3ec8" /></span>
+
+##### 5.3.1.2. When removing a staker from the pool the above addition step has to be reverted
+
+This can be done by subtracting the <img src="0005-pablo-distribution-assets/images/stem-7c4ec4f9c189cb8f3edb39740e43c33f.png" width="16" height="10" alt="stem 7c4ec4f9c189cb8f3edb39740e43c33f" /></span>
```

Comment:
> This can be done by subtracting the stem 7c4ec4f9c189cb8f3edb39740e43c33f of the staker from the total pool and reducing the pool shares accordingly.

please clarify.

when we add new stake, we add delta_p to total pool, and remove d_n from stake of each user (or new users)? but then when we remove share, we should remove d_n from pool. so we addd delta_p and remove d_n. what is relation of delta_p to d_n?